linker.go

```diff
@@ -331,8 +334,11 @@ fixups:
 				// If the kfunc call is weak and not found, poison the call. Use a recognizable constant
 				// to make it easier to debug. And set src to zero so the verifier doesn't complain
 				// about the invalid imm/offset values before dead-code elimination.
-				ins.Constant = kfuncCallPoisonBase
-				ins.Src = 0
```

Comment:
I assume zeroing Src was redundant to begin with, but just checking. Comment may need a little touch-up if that's the case.

Reply:
No, src had to change because that indicates that the call is a kfunc call. This is now taken care of by by `Call`.